src/data/modifiers.txt

```diff
@@ -8189,20 +8237,25 @@ Skill	Gift of the Maid	Meat Drop: +50
 Skill	Gift of the Priest	Spell Damage: +40
 Skill	Gift of the Scribe	Experience (Mysticality): +5
 Skill	Gluteus Maximus	Muscle: +25
+Skill	Gluttony	Experience: -2, Stomach Capacity: +2
+# Gluttony duplicates Opossum's food gains
 Skill	Gnefarious Pickpocketing	Meat Drop: +10
 Skill	Gnomish Hardigness	Maximum HP Percent: +5
 Skill	Golgi Apparatus	Absorb Stats: +25
 # Good Singing Voice: Doubles duration of Boris songs
 Skill	Gorgonzola's Guile	Item Drop: +25
 # Gourmand: Increases adventures from food
-# Gravitational Compression: +100% Item Drops from Monsters (Scales with your Mysticality)
 Skill	Gravitational Compression	Item Drop: [min(ceil(basemys/5),100)]
+# Gravitational Compression: +100% Item Drops from Monsters (Scales with your Mysticality)
+Skill	Greed	Meat Drop: +50, Item Drop: -15
+# Grogged For Battle
+Effect	Groggipopped	Stench Resistance: +1, Sleaze Resistance: +1
 # Hammer Throw: Level 1: Briefly stun your foe and reduce their stats by 10%
 # Hammer Throw: Level 2: Reduction increased to 15%
 # Hammer Throw: Level 3: Reduction increased to 20%
 Skill	Hamstrings	Initiative: +10
-Skill	[15025]Hard Drinker	Liver Capacity: +10
 Skill	[18008]Hard Drinker	Liver Capacity: +5
+Skill	[15025]Hard Drinker	Liver Capacity: +10
```

Comment:
what lead to this sort? It's an odd one. 5 before 10?

Reply:
Yeah - I totally ignore id when sorting. I want to keep it this way haha.